File: Java/Assignment19_1.java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment19_1.java
// Description : write a program which Accept n number from user and return 
//               frequency of even numbers
// input.      : n : 6
// elements.   : 85   33  3   55  93  66
// output.     : 1                        
// Author      : Aniket Devaba Markad                        
// Date        : 10/1/2026
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.Scanner;

class ArrayX
{
    int EvenFrequency(int Arr[])
    {
        int iCnt = 0,iEvenElement = 0;

        for(iCnt = 0;iCnt < Arr.length;iCnt++)
        {
            if(Arr[iCnt] % 2 == 0)
            {
                iEvenElement ++;
            }
        }
        return iEvenElement;
    }

}
class Assignment19_1
{
    public static void main(String []a)
    {
        Scanner sobj = new Scanner(System.in);

        int Arr [] ;

        System.out.println("enter the size of array :");
        int iLength = sobj.nextInt();

        Arr = new int[iLength];

        System.out.println("enter the "+ iLength + "elements in array :");

        for(int iCnt= 0; iCnt < Arr.length;iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }

        ArrayX aobj = new ArrayX();

        int iRet = aobj.EvenFrequency(Arr);

        System.out.println(" even number of element is :"+ " "+iRet);
    }
}

File: CPP/Assignment19_1.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment19_1.cpp
// Description : write a program which Accept n number from user and return 
//               frequency of even numbers.
// input       : n : 6
// elements    : 85   33  3   55  93  66
// output      : 1
// Author      : Aniket Devaba Markad                        
// Date        : 11/1/2026
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include <iostream>
using namespace std;

class ArrayX
{
public:
    int EvenFrequency(int Arr[],int iLength)
    {
        int iCnt = 0, iEvenElement = 0;

        for(iCnt = 0;iCnt < iLength;iCnt++)
        {
            if(Arr[iCnt] % 2 == 0)
            {
                iEvenElement++;
            }
        }
        return iEvenElement;
    }
};

int main()
{
    int iLength = 0;
    int *Arr = NULL;

    cout<<"enter the size of array :\n";
    cin>>iLength;

    Arr = new int[iLength];

    cout<<"enter the elements in array :\n";

    for(int iCnt = 0;iCnt < iLength;iCnt++)
    {
        cin>>Arr[iCnt];
    }

    ArrayX *aobj= new ArrayX();
    cout<<"even number of element is : "<<aobj->EvenFrequency(Arr,iLength)<<"\n";

    delete []Arr;
    return 0;
}
